keepass-mode.el

```diff
@@ -49,14 +49,28 @@
   (keepass-mode-update-group-path (mapconcat #'identity (butlast (split-string keepass-mode-group-path "/" t) 1) "/"))
   (keepass-mode-open))
 
-(defun keepass-mode-copy-password ()
-  "Copy current entry password to clipboard."
-  (interactive)
+(defun keepass-mode-copy (field)
+  "Copy current entry field to clipboard."
```

Comment:
According with eLisp style guides, the arguments in the functions documentation should be uppercase.
```suggestion
  "Copy current entry FIELD to clipboard."
```